easybuild/easyconfigs/d/DFTB+/DFTB+-17.1-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'ConfigureMake'
+
+name = 'DFTB+'
+version = '17.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.dftb-plus.info'
+description = """DFTB+ is a fast and efficient versatile quantum mechanical simulation package.  It is based on the 
+ Density Functional Tight Binding (DFTB) method, containing almost all of the useful extensions which have been
+ developed for the DFTB framework so far.  Using DFTB+ you can carry out quantum mechanical simulations like with
+ ab-initio density functional theory based packages, but in an approximate way gaining typically around two order of
+ magnitude in speed."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+# some tests, e.g. spinorbit, fail with more aggressive optimization
+toolchainopts = {'lowopt': True, 'optarch': False}
+
+source_urls = ['https://github.com/dftbplus/dftbplus/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = [
+    '1c3d90418299f3c05e0adadf7c683b41f08600bea0ecde5edbd2206198d92a00',  # 17.1.tar.gz
+    'ffbd4721a8a01326be11ef780cb392265d05631fac144a9ae6e76aceae935e2b',  # DFTB+-17.1_test_dptools.patch
+]
+
+patches = ['DFTB+-17.1_test_dptools.patch']
```

Comment:
@migueldiascosta Please move this up above `checksums`, and avoid hardcoding the version number by using `%(version)s`